dataclients/kubernetes/kube_test.go

```diff
@@ -987,15 +987,15 @@ func TestHealthcheckReload(t *testing.T) {
 			ProvideHealthcheck: true,
 		})
 
-		dc.LoadAll()
+		prevr, _ := dc.LoadAll()
 		api.failNext = true
 
 		r, err := dc.LoadAll()
 		if err != nil {
-			t.Error("failed to fail")
```

Comment:
similarly to my other comment.

Actually, i think the tests "use healthcheck, fail" and "use healthcheck, reload fail" are not necessary anymore because they were testing a behavior that was removed. At least we can fix something by purely removing code :)